pkg/k8s/apis/cilium.io/v2/register_test.go

```diff
@@ -77,3 +78,29 @@ func (s *CiliumV2RegisterSuite) TestNeedsUpdateCorruptedVersion(c *C) {
 
 	c.Assert(needsUpdate(crd), Equals, true)
 }
+
+func (s *CiliumV2RegisterSuite) TestCIDRRegex(c *C) {
+	testCIDRs := []string{
```

Comment:
Since the regex is inscrutable, is there any chance we can add some more IPv6 addresses here? in particular, longer prefixes and ones with the middle 0s filled in (I'm pretty sure that is legal and the `::` is just a convenience).

Reply:
Sure, I added a few examples. Let me know if you have any more that you'd like to test.

Reply:
I think that covers it. I just didn't want to miss a more verbose address. thanks!